src/client/App.js

```diff
@@ -39,6 +40,7 @@ function App() {
             component={ForgotPassword}
           />
           <PrivateRoute exact path="/dashboard" component={Dashboard} />
+          <PrivateRoute exact path="/addbatch" component={AddBatch} />
```

Comment:
I would write the path as `"/add-batch"` with the dash between the words.

Reply:
Yes please use add-batch.

It's already working like that on the side bar menu

![image](https://user-images.githubusercontent.com/6642037/80010727-1f810c80-84cb-11ea-98ea-317277c39092.png)